batavia/batavia.js

```diff
@@ -6,6 +6,16 @@ BigNumber.config({
     ROUNDING_MODE: BigNumber.ROUND_HALF_EVEN
 })
 
+// Configure logging.
+if (process.env.PYBEE_DEBUG && process.env.PYBEE_DEBUG !== '0') {
```

Comment:
BATAVIA_DEBUG would be a better variable name - referring the project, rather than the meta-project.